SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -141,7 +144,7 @@ public string LoadAndFixFile(string sarifPath, ILogger logger)
 
                 File.WriteAllText(newSarifPath, changedSarif);
 
-                logger.LogInfo(Resources.MSG_SarifFixSuccess);
+                logger.LogInfo(Resources.MSG_SarifFixSuccess, newSarifPath);
```

Comment:
These other changes are tweaks to make the logging more useful e.g. it's easier to read a message saying the file is valid than to infer it from the absence of a message.